doc/main.dox

```diff
@@ -96,7 +96,7 @@ sockperf: [SERVER] listen on:
 [ 2] IP = 192.168.161.129 PORT =  7003 # TCP
 [ 3] IP = 192.168.161.129 PORT =  7004 # TCP
 sockperf: Warmup stage (sending a few dummy packets)...
-sockperf: [tid 4857] using select() to block on socket(s)
+sockperf: [tid 4857] using epoll() to block on socket(s)
```

Comment:
this edit is a mistake, because the command line above it says: 
  $bin/sockperf server -f conf.file **-F s**